### ISSUE TITLE
Unable to view contents of encrypted database

### ISSUE THREAD

**Issue by yitchoong** [couchbaselabs/CouchbaseLiteViewer#22]
Please consider an enhancement to CouchbaseLite Viewer to allow for viewing the contents in an encrypted database.

**Comment by RubhaK** [comment]
Yes. I am also facing this issue. Its been mandated by our client to encrypt the database. Do we have any solution for this issue now?
